File: 백준/Bronze/34071. 첫 번째 문제는 정말 쉬운 문제일까？/첫 번째 문제는 정말 쉬운 문제일까？.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int n,maxn=0,minn=100;
    cin>>n;
    vector<int> arr(n);
    for(int i=0;i<n;i++) {
        cin>>arr[i];
        if(maxn<arr[i])
            maxn=arr[i];
        if(minn>arr[i])
            minn=arr[i];
    }
    cout<<(minn==arr[0]?"ez":(maxn==arr[0]?"hard":"?"));
    return 0;
}

File: 백준/Bronze/34071. 첫 번째 문제는 정말 쉬운 문제일까？/첫 번째 문제는 정말 쉬운 문제일까？.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int maxn=0,minn=100,n = Integer.parseInt(scan.nextLine());
        int []arr=new int[n];
        for(int i=0;i<n;i++) {
            arr[i] = Integer.parseInt(scan.nextLine());
            if(maxn<arr[i])
                maxn=arr[i];
            if(minn>arr[i])
                minn=arr[i];
        }
        System.out.println(minn==arr[0]?"ez":maxn==arr[0]?"hard":"?");
    }
}